src/Actions/BcgovSettings.php

```diff
@@ -122,6 +139,13 @@ public function bcgov_register_settings() {
             [ $this, 'sanitize_bcgov_google_site_name_settings' ]
         );
 
+        // Add the Documentation URL setting.
+        register_setting(
+            'bcgov-settings-group',
+            'bcgov_documentation_url_settings',
+            [ $this, 'sanitize_bcgov_documentation_url_settings' ]
```

Comment:
are sure you can't just do this 
```suggestion
            'sanitize_text_field'
```

Reply:
Sure. I was following the existing settings as an example. Seems like a good simplification.